ecs-cli/modules/utils/utils.go

```diff
@@ -106,3 +107,40 @@ func GetPartition(region string) string {
 		return "aws"
 	}
 }
+
+// ParseLoadBalancers returns a list of load balacners struct
+// For users wanting to register multiple or one load balancers to a single service
+func ParseLoadBalancers(flagValue []string, list []*ecs.LoadBalancer) ([]*ecs.LoadBalancer, error) {
+
+	for i := 0; i < len(flagValue); i++ {
+		m := make(map[string]string)
+		targetGroupArn := false
+		keyValPairs := strings.Split(flagValue[i], ",")
+
+		for _, kv := range keyValPairs {
+			pair := strings.SplitN(kv, "=", 2)
+			m[pair[0]] = pair[1]
+			if pair[0] == "targetGroupArn" {
+				targetGroupArn = true
+			}
+		}
+		containerPort, err := strconv.ParseInt(m["containerPort"], 10, 64)
```

Comment:
Would be better to use `ParseUint` since port number >= 0.

Reply:
I tried to convert it as a ParseUint however in the api.go file, it is listed as a int64. I tried to change it to a uint64 in that api.go but it warns me to not edit this file as it is generated so I didn't. We can keep it to a int64 unless you have other ideas?

Reply:
oh ok then just deep it to int64 :)